README.md

```diff
@@ -169,14 +172,31 @@ The command above will create three pods associated to the existing `imagestream
 objects that point to `openstack`, `openshift` and `kubevirt` must-gather container
 images.
 
-```
+```bash
 [must-gather] OUT pod for plug-in image quay.io/openstack-k8s-operators/openstack-must-gather:latest created
 [must-gather] OUT pod for plug-in image quay.io/kubevirt/must-gather@sha256:501e30ac7d5b9840a918bb9e5aa830686288ccfeee37d70aaf99cd2e302a2bb0 created
 [must-gather] OUT pod for plug-in image quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:e9601b492cbb375f0a05310efa6025691f8bba6a97667976cd4baf4adf0f244c created
 ...
 ...
 ```
 
+### EDPM compute must gather via `sos` tool
+
+To gather config files and logs from the deployed compute nodes, we need to rely on the `sos` tool (former `sosreport`).
+
+There's an open [PR](https://github.com/sosreport/sos/pull/3414) to enable path gathering from EDPM deployed compute nodes so if you want to test it on running nodes, you need to follow these steps on a running compute node:
+
+```bash
+cd ~
+git clone https://github.com/rebtoor/sos.git -b osp18
```

Comment:
As long as [1] merges, you can git clone `https://github.com/sosreport/sos` on the main branch

[1] https://github.com/sosreport/sos/pull/3414

Reply:
I guess that we should remove the clone step, since we expect that sos will be already installed on compute nodes. WDYT?

Reply:
+1, and we can provide instructions to run the sosreport both from a coreos node and from an edpm one using the new profile